src/corppa/poetry_detection/annotation/command_recipes.py

```diff
@@ -0,0 +1,87 @@
+from collections import Counter, defaultdict
+
+from prodigy.components.db import connect
+from prodigy.core import Arg, recipe
+from prodigy.errors import RecipeError
+from prodigy.util import SESSION_ID_ATTR, msg
+
+
+@recipe(
+    "page-stats",
+    dataset=Arg(help="Prodigy dataset ID"),
+)
+def ppa_stats(dataset: str) -> None:
+    # Load examples
+    DB = connect()
+    if dataset not in DB:
+        raise RecipeError(f"Can't find dataset '{dataset}' in database")
+    examples = DB.get_dataset_examples(dataset)
+    n_examples = len(examples)
+    msg.good(f"Loaded {n_examples} annotations from {dataset} dataset")
+
+    # Get stats
+    examples_by_page = Counter()
+    examples_by_session = defaultdict(list)
+    for ex in examples:
+        # Skip examples w/ answer or (page) id
```

Comment:
by "w/" here did you mean "w/o" / without?
```suggestion
        # Skip examples without answer or (page) id
```

Reply:
Good catch!